### ISSUE TITLE
Cannot make Sun to work, neither Jellyfish

### ISSUE THREAD

**Issue by Tudumanu** [Gamua/Starling-Extension-Particle-System#30]
Hi Daniel,

I've tried to see all the Demo examples, but Sun doesn't work at all and Jellyfish works with squares (see the image below):
![bug-jellyfish](https://user-images.githubusercontent.com/5518496/34921886-26026c5a-f96f-11e7-9389-0aec50393ffa.PNG)

 I got **sun to work only when changed the .pex file blendFunc**:

```
//FROM
  <blendFuncSource value="772"/>
  <blendFuncDestination value="771"/>

//TO
  <blendFuncSource value="770"/>
  <blendFuncDestination value="772"/>
```

Image of sun "working" with the changes in blendFunction:
![sun](https://user-images.githubusercontent.com/5518496/34921916-99dab4a2-f96f-11e7-94ff-272f10e7f4d3.PNG)

I am using FD to compile the project with Flex + AIR 4.6.0 + 23.0.0, compiled the project with `-swf-version=34`, using FP sa debugger (latest version: 28) and Starling v2.1 that says `[Starling] Context ready. Display Driver: DirectX11 (Baseline Extended)`. Any ideas of what I'm missing?

**Comment by Tudumanu** [comment]
linking to Daniel: @PrimaryFeather

**Comment by PrimaryFeather** [comment]
_tl;dr:_ The following code should fix the issue:

```as3
stage.color = 0xff000000;
```

In the past, Starling's background had an alpha value of `1.0` per default, but this was [counter-intuitive](https://forum.starling-framework.org/topic/blend-mode-is-not-working-as-expected#post-97488) for some users, so I changed the default to `0.0`. This, however, changes the output of some blend factor combinations (i.e. blend modes). The line above changes the `alpha` with which the back buffer is cleared to `1.0`, and thus restores the original behavior.

Please let me know if that fixe this issue for you! (If it doesn't, please also try upgrading to the latest 2.3 version of Starling.)

**Comment by PrimaryFeather** [comment]
(I just added that information to the particle system's wiki page, too.)

**Comment by Tudumanu** [comment]
@PrimaryFeather **If you see on the init() function of your demo code, this is already defined on line 58** `stage.color = 0xff000000;`. removing this line I've got this on sun and jellyfish:
![sun](https://user-images.githubusercontent.com/5518496/34955126-3556b9e0-fa0a-11e7-9103-4c3491de80eb.PNG)

![jellyfish](https://user-images.githubusercontent.com/5518496/34955131-3932aee8-fa0a-11e7-83ae-05a2e75e5313.PNG)

**Comment by Tudumanu** [comment]
Adding a Quad on the line I've got this:
![sun-blacksquare](https://user-images.githubusercontent.com/5518496/34955260-c5170a30-fa0a-11e7-8d21-01e111b5fec3.PNG)

![jellyfish-blacksquare](https://user-images.githubusercontent.com/5518496/34955263-c8e9ba5e-fa0a-11e7-817f-a30ecca22fa1.PNG)

**Comment by Tudumanu** [comment]
Testing with Starling 2.3, **with or without the line** `stage.color = 0xff000000;` I've got the same behaviour as the version 2.1 adding the Quad. I think you could remove the information of the stage.color and add the information to use Starling 2.3 😛 

Thanks Daniel!

**Comment by PrimaryFeather** [comment]
Hmhm, that's weird, then something else must have changed that caused compatibility issues. But I'm glad to hear it works now! :smile:

**Comment by PrimaryFeather** [comment]
(Oh, and yes: quad behind the object will also fix those blend mode issues!)
